plugin/src/test/groovy/com/novoda/buildproperties/internal/FilePropertiesEntriesTest.groovy

```diff
@@ -10,11 +9,12 @@ import static org.junit.Assert.fail
 
 class FilePropertiesEntriesTest {
 
-    private static final File PROPERTIES_FILE = new File(Resources.getResource('any.properties').toURI())
+    private static
```

Comment:
⚠️ why the `\n` here?

Reply:
All of sudden I have the same issue with IntelliJ, it seems to happen when the line is too long, we might need to include the code styles in the repo to avoid issues like this

Reply:
Did anyone say including the code style in the repo?

![](https://i.imgur.com/ESPvKGb.gif)

Ping me if you need any help.